tests/test_fastpdb.py

```diff
@@ -107,8 +107,6 @@ def test_get_structure(path, model, altloc, extra_fields, include_bonds):
     
     if ref_atoms.box is not None:
         assert np.allclose(test_atoms.box, ref_atoms.box)
```

Comment:
With this adaption, the test would not only check if no exception appears but also that the box has the proper `None` value. I think the previous `assert test_atoms == None` was just a typo that never surfaced, as there was no test case where there is no box.

```suggestion
    if ref_atoms.box is not None:
        assert np.allclose(test_atoms.box, ref_atoms.box)
    else:
        assert test_atoms.box == None
```

Reply:
Great catch, I modified the original commit to make this change!